docs/guide.md

```diff
@@ -71,20 +71,104 @@ arrow keys to navigate through the list. If you accidentally select an option
 you don't want, simply use the left arrow key to deselect it.
 
 The following is a description of the options in the TUI.
+
 ## The author of project
 
-- **author_name**
-- **author_email**
+With SciCookie, you can easily add information about the author of the
+project by filling in the **author_full_name** and **author_email** fields,
+which refer to the name and email address of the person or organisation
```

Comment:
@xmnlab Are we going to use British English as the standard or American English? 🤔

Reply:
Thanks @Anavelyz

Reply:
I personally prefer the American English, do you have any preference?